python-sdk/pyproject.toml

```diff
@@ -62,7 +62,7 @@ google = [
 ]
 snowflake = [
     "apache-airflow-providers-snowflake",
-    "snowflake-sqlalchemy>=1.2.0,<=1.2.4",
+    "snowflake-sqlalchemy>=1.2.0",
```

Comment:
Not sure why this was restricted but because of this when we are installing `astro-sdk` with airflow constraints file getting below error
```
The conflict is caused by:
    astro-sdk-python[amazon,google,postgres,snowflake] 1.0.0 depends on snowflake-sqlalchemy<=1.2.4 and >=1.2.0; extra == "snowflake"
    The user requested (constraint) snowflake-sqlalchemy==1.4.2
```

Reply:
related to https://github.com/astronomer/astro-sdk/pull/1185 cc @utkarsharma2

Reply:
In #1185  look like we have changed `SQLAlchemy` only so `snowflake-sqlalchemy` version was restricted even before

Reply:
Also look like import issue has been fixed in latest release https://github.com/snowflakedb/snowflake-sqlalchemy/releases/tag/v1.4.3

Reply:
Cool. Could you remove the version restriction there then as well? I think there is then no reason to limit that version.